inference-engine/src/low_precision_transformations/src/common/network_helper.cpp

```diff
@@ -951,14 +951,38 @@ NetworkHelper::InsertDequantizationResult NetworkHelper::moveDequantizationAfter
         parent = std::make_shared<DequantizationConvert>(parent, dequantization.convert->get_output_element_type(0));
         ngraph::copy_runtime_info({ newOperation, parent }, parent);
     }
+
     if (moveSubtract && (dequantization.subtract != nullptr)) {
         auto subtractConstant = dequantization.subtract->get_input_node_shared_ptr(1);
-        parent = std::make_shared<DequantizationSubtract>(parent, subtractConstant);
+        const element::Type parentPrecision = parent->get_output_element_type(0);
+        if (parentPrecision.bitwidth() < subtractConstant->output(0).get_element_type().bitwidth()) {
+            THROW_IE_LPT_EXCEPTION(*parent) <<
```

Comment:
Why cannot we insert convert for this case as well?

Reply:
was added in issue #42932